examples/proof-of-concept/rabbitmq/things/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.14-buster
+
+WORKDIR /root/thing
+
+# Create and install the cert
```

Comment:
I guess this will only work if you manually added the cert to k8s, unless that's done automatically somewhere in the PR that I've missed?

Reply:
Great spot. I think caching stopped this issue showing up (I changed this last). Reverted to using a pre created certificate.